tools/serverpod_cli/lib/src/test_util/builders/enum_definition_builder.dart

```diff
@@ -96,4 +98,9 @@ class EnumDefinitionBuilder {
     _documentation = documentation;
     return this;
   }
+
+  EnumDefinitionBuilder withType(TypeDefinition type) {
+    _type = type;
+    return this;
+  }
```

Comment:
Suggestion: Remove this method you should never really set this manually, instead add an internal type definition construction if someone changes the enum definition class name.

Apply the same logic for the class definition builder too.